README.md

```diff
@@ -1,4 +1,186 @@
-# Website
+# techwriter.pl
+
+This is a blog for technical writers, created using
+[Docusaurus](https://docusaurus.io).
+
+Notable topics below:
+
+- Creating a post
+- Setting up a GitHub codespace
+- Setting up VS Code in your codespace
+
+## Creating a new post
+
+Your post is a Markdown file with a `.md` extension, usually along with some
+graphics. The post is identified by a title, date, author, and tags.
+
+**Prerequisite:** Before you add your first post, set up your codespace and VS
+Code. See the relevant sections below.
+
+To add a post:
+
+0. Create a branch. You may be working from a GitHub issue. If so,
+   [read about creating branches here](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-a-branch-for-an-issue).
+
+   If you are not working form a GitHub issue (why not? 🤷), you can manually
```

Comment:
Replace "working form" with "working from".

A stylistic note on the conversational bits like "why not?"- it reads ok the first time, it does not read as well if you have to re-read the section. You could say that there is the preferred way - starting with a Github issue - and the acceptable way - manual branch creation.